src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -90,6 +95,11 @@ public class OrganelleDefinition : IRegistryType
     /// </summary>
     public OrganelleFeatureTag[] FeatureTags = Array.Empty<OrganelleFeatureTag>();
 
+    /// <summary>
+    ///   Description of processes the organelle performs
+    /// </summary>
+    public string? ProcessesDescription = null;
```

Comment:
The default value here being null is unnecessary. Also processes description is more important for an organelle than the description in my opinion. But both of these should be checked in the Check method that they aren't null and aren't empty.

Reply:
What about unimplemented organelles that don't have processes description?

Reply:
I guess that's a fine exception to add to the code, but otherwise the condition should be enforced.